### DISCUSSION TITLE
How to use VisitMutAstPath with Program.fold_with() ?

### DISCUSSION THREAD

**Discussion by GeeekyBoy** [swc-project/swc#6944] [Q&A]
I'm trying to access parent from child node like I do in babel's (path.parent).
One way I think I can follow to implement a similar functionality is to implement VisitMutAstPath instead of VisitMut. The problem here is that Program.fold_with() doesn't accept VisitMutAstPath.
Is there any code samples that shows how VisitMutAstPath can be used ?

**Comment by kdy1** [comment]
IIRC there's `visit_mut_with_ast_path`. You can use `cargo doc`, too.
